client-js/app/elements/labrad-plot.ts

```diff
@@ -367,8 +367,18 @@ export class Plot extends polymer.Base {
 
     this.lastData = null;
 
+    this.dataLimits.xMin = 0;
+    this.dataLimits.xMax = 0;
+    this.dataLimits.yMin = 0;
+    this.dataLimits.yMax = 0;
```

Comment:
Existing 2D_multi data test is sufficient to test the 2D case. It elucidated the fact the z-axis wasn't being adjusted correctly when all values were equal, so I fixed that also.